### ISSUE TITLE
WebGL: INVALID_ENUM: activeTexture: texture unit out of range

### ISSUE THREAD

**Issue by undertowe** [avgjs/pixi-live2d#4]
I'm getting the following warning in the console when running pixi-live2d:

WebGL: INVALID_ENUM: activeTexture: texture unit out of range

This happens the first time Live2DSprite._renderWebGL() gets called, but not on successive calls. The line that triggers the warning:

gl.activeTexture(gl.TEXTURE0 + _activeTextureLocation);

It appears that the value of _activeTextureLocation is still 999 (initial value?) on the first call. I've tested with both haru and epsilon models. Here's a demo of the issue:

https://undertowe.github.io/pixi-live2d/

Interestingly, I don't see this warning on the [example](https://avgjs.github.io/pixi-live2d-example/). In that example the value of _activeTextureLocation is 0 on the first call to _renderWebGL(). My code is using the same versions of the Live2D SDK (2.1.00_1) and pixi.js (4.0.3) as the example, so I'm not sure what's causing the difference in behavior.

Any ideas?

**Comment by undertowe** [comment]
So, I think I figured out why this is happening in my [test](https://undertowe.github.io/pixi-live2d/) but not the [example](https://avgjs.github.io/pixi-live2d-example/). In the example you're creating a PIXI.Sprite (of the background) before creating the Live2DSprite of the model. In my code I'm creating only the Live2DSprite of the model. I just tried creating a Sprite first and the warning went away. Without delving into the code, it seems that creating a Sprite causes the value of _activeTextureLocation to be set properly, but that doesn't happen when you create a Live2DSprite.

I'll try to poke around and figure out what needs to happen when creating a Live2DSprite in order to prevent this warning.
